### PR TITLE
Added configs for layers in the Map Config

### PR DESCRIPTION
**Pull request by alonakos**
`showOpacitySlider`

### CODE REVIEW

**Review by robyngit — CHANGES_REQUESTED** [review]
Congrats on your first PR!!! 🎉 Thanks so much for working on this! Everything looks great, just need to add some documentation and resolve linting/formatting issues. Details about fixing the linting and formatting errors can be found [here](https://github.com/NCEAS/metacatui/blob/main/CONTRIBUTING.md#-code-style).

File: src/js/models/maps/assets/MapAsset.js [hunk 1]
```diff
@@ -130,6 +130,7 @@ define([
           status: null,
           statusDetails: null,
           hideInLayerList: false,
+          showOpacitySlider: true,
```
[the remark this answers is not in the document]
Comment (alonakos) [reply src/js/models/maps/assets/MapAsset.js:135]: Thank you! I just added jsdoc comment in MapAssest. Do you think this looks good?

File: src/js/views/maps/LayerDetailsView.js [hunk 2]
```diff
@@ -217,11 +217,18 @@ define([
...
+          // Remove and do not render opacity section if showOpacitySlider is false
+          if (model.get("showOpacitySlider") === false) {
+            this.renderedSections = this.renderedSections.filter(
+              (item) => item.label !== "Opacity",
+            );
+          }
+
           this.renderedSections.forEach(function (section) {
-            var detailSection = new LayerDetailView({
+            const detailSection = new LayerDetailView({
```
Comment (alonakos) [inline src/js/views/maps/LayerDetailsView.js:228]: I implemented the changes on line 228, line 231 following the reviewdog suggestions.

File: src/js/models/maps/assets/MapAsset.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (robyngit) [reply src/js/models/maps/assets/MapAsset.js:135]: Looks great! 👍🏻

File: src/config/config.js
Comment (robyngit) [inline src/config/config.js:1]: Oops! I think you commit config/config.js by accident (very easy to do)!

Comment (alonakos) [reply src/config/config.js:1]: Oops, thank you for pointing it out! Fixed

**Review by robyngit — APPROVED** [review]
🎉 Excellent!!!

**Pull request merged**